### ISSUE TITLE
Fix media with apostrophe in the title failing

### ISSUE THREAD

**Issue by SpaceK33z** [SpaceK33z/plex2netflix#3]
Apparently this doesn't work.

**Comment by onedr0p** [comment]
+1

**Comment by SpaceK33z** [comment]
@onedr0p, could you maybe provide a move title where this fails with? Because so far all my movies with apostrophes are working.

**Comment by onedr0p** [comment]
**This is stuff that gets a 404**
```
"Everybody Loves Raymond" The Bachelor Party (TV ... (undefined) - failed request (code: 404)
Hans Brinker or the Silver Skates: Part 1 (1962) - failed request (code: 404)
American Hero (2015) - failed request (code: 404)
```

**This are apostrophe errors**
```
Child's Play 3 (1991) - failed request (code: Error: invalid response)
Child's Play (1988) - failed request (code: Error: invalid response)
Child's Play 2 (1990) - failed request (code: Error: invalid response)
A Bug's Life (1998) - failed request (code: Error: invalid response)
```

**Comment by garrettben** [comment]
Ferris Bueller's Day Off (1986) - failed request (code: Error: invalid
response)
Harry Potter and the Philosopher's Stone (2001) - failed request (code:
Error: invalid response)
It's Bad for Ya (2008) - failed request (code: Error: invalid response)
Kelly's Heroes (1970) - failed request (code: Error: invalid response)
The King's Speech (2010) - failed request (code: Error: invalid response)
On Her Majesty's Secret Service (1969) - failed request (code: Error:
invalid response)
Pan's Labyrinth (2006) - failed request (code: Error: invalid response)
One Flew Over the Cuckoo's Nest (1975) - failed request (code: Error:
invalid response)


Might be the combination of apostrophe and parenthesis.

**Comment by Septan** [comment]
Let's Be Cops (2014) - failed request (code: Error: invalid response)
Scooby-Doo! And Kiss: Rock and Roll Mystery (2015) - failed request (code: 404)
Three O'Clock High (1987) - failed request (code: Error: invalid response)

**Comment by SpaceK33z** [comment]
Unfortunately I can't fix the `Error: invalid response` for now, because this is a bug in the Netflix Roulette API. I will look into the `failed request (code: 404)` ones.

Does someone happen to know another API that allows you to use the IMDb ID to check the Netflix availability?

**Comment by SpaceK33z** [comment]
I debugged the movie `Scooby-Doo! And Kiss: Rock and Roll Mystery`. The tool finds it with imdb id tt4717798, which is [correct](http://www.imdb.com/title/tt4717798/). It then tries to do a GET request to `https://netflixroulette.net/api/v2/usa/imdb/?imdbId=tt4717798`, which fails with a 404. I reported the issue in Codeusa/Netflix-Roulette-API#11

**Comment by SpaceK33z** [comment]
Fix released with `plex2netflix@0.5.0`. Better late than never!
